### ISSUE TITLE
Swag should not generate document of private fields

### ISSUE THREAD

**Issue by comphilip** [swaggo/swag#648]
**Describe the bug**

Swag should not generate document of private fields

**Steps to reproduce the behavior**

With input struct
```go
type NextTradeServer struct {
	IP    string                `gorm:"column:ip;primary_key" json:"ip"`
	Extra json.RawMessage       `gorm:"column:extra" json:"extra"`
	cache *NextTradeServerExtra // private field
}
```
Got result:
```yaml
  model.NextTradeServer:
    properties:
      cache:
        $ref: '#/definitions/model.NextTradeServerExtra'
        type: object
      extra:
        type: string
      ip:
        type: string
    type: object
  model.NextTradeServerExtra:
    properties:
      start_times:
        items:
          type: string
        type: array
      stop_times:
        items:
          type: string
        type: array
      trader_ids:
        items:
          type: integer
        type: array
    type: object
```

**Expected behavior**

Private fields should not be document generated.

**Your swag version**

swag version v1.6.5

**Your go version**

go version go1.14 linux/amd64

**Workground**

Add `json:"-"` to private field will stop swag generate documents.

Side effect: go lint reports `struct field cache has json tag but is not exported`

Replace `json:"-"` with `swaggerignore:"true" does not work, lint warning gone but swagger document generated.

**Comment by affanshahid** [comment]
Having the exact same problem. `swaggerignore:"true"` does not seem to be working

**Comment by sdghchj** [comment]
Feature of swaggerignore was merged into master branch just a few days ago, after v1.6.5.
Try master,please.

**Comment by comphilip** [comment]
@sdghchj For private field, swaggerignore is not needed

**Comment by sdghchj** [comment]
I really appreciate your unconstrained idea to constrain that private fields shouldn't be generated.
But, compatibility needs to be considered for those who already using it in that way, and how if someone in future come and ask why not generate private fields?
I still think 'private' is a concept in golang's respect, and tag is a better way for this case, more flexible.

**Comment by comphilip** [comment]
@sdghchj Agree with you. Then swaggerignore in next version after v1.6.5 will fixed this issue.

**Comment by sdghchj** [comment]
@comphilip  Given that json in golang sdk indeed excludes private fields, I think you're right. 
I'm not sure about Gin's unmarshaling params in query

**Comment by sdghchj** [comment]
Have fixed by #675
